pupil_src/capture/run_capture.bat

```diff
@@ -1,2 +1,2 @@
-PATH %PATH%..\..\pupil_external
+SET PATH=%PATH%;%~dp0..\..\pupil_external
```

Comment:
PATH %PATH%%~dp0..\..\pupil_external

is not working for me.

I think that `SET` just temporary adds the path to the `PATH` variable. I thought this is what you want to achieve with it?

When you call 

    echo %PATH%
you see that there is no `;`

I'm using Windows 10 64 Bit.